### ISSUE TITLE
RPC documentation outdated

### ISSUE THREAD

**Issue by nearmax** [nearprotocol/docs#48]
Start a local node with `myaccount` account as a validator.
Issue a transaction:
```
http post http://127.0.0.1:3030/ jsonrpc=2.0 method=query params:="[\"account/myaccount\",[]]" id="dontcare"
```

Observe error message:
```
HTTP/1.1 200 OK
content-length: 156
content-type: application/json
date: Fri, 26 Jul 2019 18:39:40 GMT

{
    "error": {
        "code": -32602,
        "data": "Failed parsing args: invalid type: sequence, expected a string",
        "message": "Invalid params"
    },
    "id": "dontcare",
    "jsonrpc": "2.0"
}

```

**Comment by ilblackdragon** [comment]
Reopening this, as it's true again.
Query, block, tx status and missing new RPCs like valiators, network_info and chunk

**Comment by amgando** [comment]
this seems to have been resolved by #164
